main.ts

```diff
@@ -61,13 +67,33 @@ async function run(): Promise<void> {
       }
     }
 
-    const mingw = architecture === 'i686' ? 'MINGW32' : 'MINGW64'
+    let mingw = ''
+    let extraPaths: string[] = []
+
+    switch (architecture) {
+      case 'i686':
+        mingw = 'MINGW32'
+        break
+      case 'x86_64':
+        mingw = 'MINGW64'
+        break
+      case 'aarch64':
+        mingw = 'CLANGARM64'
+        // Some binaries aren't available yet in the /clangarm64/bin folder, but Windows 11 ARM64
+        // has support for x64 emulation, so let's add /mingw64/bin as a fallback.
+        extraPaths = ['/mingw64/bin']
```

Comment:
I was specifically missing `git.exe` and friends. This fallback will no longer be needed once a first version of Git for Windows ARM64 has been released.